src/m_misc.h

```diff
@@ -28,6 +28,7 @@
 
 #ifdef _WIN32
 wchar_t *M_ConvertUtf8ToWide(const char *str);
+char *ConvertWideToUtf8(const wchar_t *wstr);
```

Comment:
Since this is no longer static, document that the return value must be freed by caller.

Moving this should be in its own commit just in case this PR needs to be reverted.

Reply:
Separated into https://github.com/chocolate-doom/chocolate-doom/pull/1598/commits/db56db1e799867bd78bd8716a0680322f42fe72a and https://github.com/chocolate-doom/chocolate-doom/pull/1598/commits/d126056cd725759eaf925d88c4e871743a6e8109